conda/cli/activate.py

```diff
@@ -110,7 +134,7 @@ def main():
         raise ValueError("unexpected command")
 
     for path in os.getenv('PATH').split(os.pathsep):
-        if path != binpath:
+        if path not in binpath:
```

Comment:
I found another problem here: for some reason `activate` adds the path as `C:\...` (capital drive letter), whereas `deactivate` tries to remove `c:\...` (lower case drive letter). So, no match is found and no path is actually removed. Since the Windows file system is (still, IIRC) case-insensitive, a case-insensitive comparison should be used on Windows here.